app/presenters/register_api/single_application_presenter.rb

```diff
@@ -34,15 +34,11 @@ def as_json
             english_language_qualifications: application_form.english_language_qualification_details,
             other_languages: application_form.other_language_details,
             disability_disclosure: application_form.disability_disclosure,
-            gender: equality_and_diversity_data['sex'],
```

Comment:
Didn't we decide to keep this or was there an update around just using HESA instead?